driver/conv_driver.hpp

```diff
@@ -490,6 +531,53 @@ int ConvDriver<Tgpu, Tref>::ParseCmdLineArgs(int argc, char* argv[])
     return 0;
 }
 
+
+template <typename Tgpu, typename Tref>
+void  ConvDriver<Tgpu, Tref>::ValidateLayoutInputParameters(std::string layout_value)
+{
+     if ((ChkLayout_ShortName()))
+     {
+        std::cerr << " Invalid Layout Short Name = " << ChkLayout_ShortName() << std::endl;
+        exit(EXIT_FAILURE);
+     }
+     else
+     {
+        if((layout_value.compare("NCHW") == 0) ||
+          (layout_value.compare("NHWC") == 0) ||
+          (layout_value.compare("NCDHW") == 0) ||
+          (layout_value.compare("NDHWC") == 0))
+          {
+            //do nothing,Values are matching as defined in Lib.
+          }
+          else
+          {
+            std::cerr << "Invalid Layout Parameter Value - " << layout_value <<  std::endl;
+            exit(EXIT_FAILURE);
+          }
+    }
+}
+
+
+template <typename Tgpu, typename Tref>
+int ConvDriver<Tgpu, Tref>::ChkLayout_ShortName()
+{
+    //check for short name of layout type
+    if ( (inflags.FindShortName("in_layout") == 'I') && 
+         (inflags.FindShortName("out_layout") == 'O') &&
+         (inflags.FindShortName("fil_layout") == 'f'))
```

Comment:
What is the reason of checking this? https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1316#discussion_r767329529 seems unresolved.

Reply:
@atamazov - Short names are part of command line argument list to MIOpen driver. This particular check is to make sure , driver has received the valid short names. Let me knw.